www/src/pages/ecosystem.js

```diff
@@ -1,28 +1,114 @@
 import React, { Component } from "react"
 import Helmet from "react-helmet"
+import { graphql } from "gatsby"
```

Comment:
Looks like a couple of unused imports in this file, too.
- "react-helmet"
- "../utils/typography"
- "../utils/presets"
- "../components/layout/page-heading"